@xen-orchestra/backups/_VmBackup.js

```diff
@@ -398,6 +404,51 @@ class VmBackup {
     this._fullVdisRequired = fullVdisRequired
   }
 
+  async _healthCheck(adapter) {
+    const settings = this._settings
+    settings.healthCheck = {
+      srUuid: '86a9757d-9c05-9fe0-e79a-8243cb1f37f3',
+      tags: [],
+    }
+    if (this._metadataFileName === undefined) {
+      return
+    }
+
+    if (settings.healthCheck === undefined) {
+      return
+    }
+
+    // @todo check if current VM has tags
+    const { tags } = this.vm
+    const intersect = settings.healthCheck.tags.filter(t => tags.includes(t))
+
+    if (settings.healthCheck.tags.length !== 0 && intersect.length === 0) {
+      return
+    }
+    let restoredVm
+    const xapi = this._xapi
```

Comment:
here we must use a xapi that can access to the SR, which is not the same as the xapi of the source VM